test/unit/CommandLineController.test.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Tests for the Command-Line Argument Parser
+ */
+
+import * as cli from '../../src/Controller/CommandLineController'
```

Comment:
Very nitpicky, but shouldn't the import go before the test documentation? I'm actually not sure if that's the convention, but it just seems right to me haha.

Reply:
Quick nitpick, but quick fix ;)